proto/injective/exchange/v2/query.proto

```
@@ -168,6 +168,18 @@ service Query {
   }
 
   // Retrieves a spot market's mid-price
+  // Projects the output of a spot swap for an exact input amount
+  rpc SpotSwapOutput(QuerySpotSwapOutputRequest)
+      returns (QuerySpotSwapOutputResponse) {
+    option (google.api.http).get = "/injective/exchange/v2/spot/swapOutput";
```

Buf (1.72.0)

[error] cannot find `google.api.http` in this scope

(COMPILE)